### PR TITLE
Add coverlet

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
I used https://github.com/up-for-grabs/up-for-grabs.net#use-the-yeoman-generator

cc: @tonerdo @petli

### CODE REVIEW

File: _data/projects/Coverlet.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Coverlet
+desc: Cross platform code coverage for .NET
+site: https://github.com/tonerdo/coverlet
+tags:
+- cross-coverage
+- dotnet-core
```
Comment (ritwik12) [inline _data/projects/Coverlet.yml]: @MarcoRossignoli Please use Use `.net core` instead of dotnet-core

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Coverlet.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- dotnet-cli
+- unittest
+- coverage
+- cross-platform
+- coverage-library
+- dotnet
+- coverlet
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/tonerdo/coverlet/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
```
Comment (ritwik12) [inline _data/projects/Coverlet.yml]: @MarcoRossignoli Please change the link to https://github.com/tonerdo/coverlet/labels/up-for-grabs

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Coverlet.yml (hunk 1 above)
Comment (MarcoRossignoli) [reply _data/projects/Coverlet.yml]: done

File: _data/projects/Coverlet.yml (hunk 2 above)
Comment (MarcoRossignoli) [reply _data/projects/Coverlet.yml]: done

**Review by ritwik12 — APPROVED** [review]

**Pull request merged**